index.md

```diff
@@ -1,11 +1,244 @@
+This was compounded by the Data Services Lead failing to understand until late in the game that she needed to be exerting oversight of the entire data release "pipeline", rather that "just" leading RSP service development and data publication.
+
+**Recommendation:**  A "shipping list" is drawn for each of the above six transfer points. The team passing the bucket has the responsibility to check that all the items in the shipment can be checked off. These shipping lists should be templated in some way (Github, JIRA) to be re-used for subsequent releases.
+
+**Recommendation:** Future data release schedules include explicit deadlines for each of those steps, with steps 1-3 being parallelized to the extent possible. Missing these early deadlines should trigger a corresponding schedule delay.
+
+**Recommendation:** A person who is _not_ the Data Services Lead, ideally from Pipelines or V&V should exercise oversight on steps 1-3 above and hand over to the Data Services Lead for steps 4-6.
+
+### Gated approval from a person/body with sign-off powers
+
+While the peer-to-peer bucket-brigade model is well suited for technical handover, there were a number of times where the Data Services Lead, specifically, had to make decisions that were not properly in her purview.
+An example of this was which HiPS product to use for the coverage map, which field to center the coverage map to, etc.
+
+Additionally, it was unclear who could be negotiated with for discussions along the lines of "do you want to go live like this, or do you want us to slip and fix this first".
+The RSP product owners provided great input and context, however they did not have the authority to slip a community-promised date either.
+Lines of command are further blurred by the fact that a data release crosses the boundaries of Data Management and System Performance.
+
+**Recommendation:** A "decider of last resort" with the expertise and organisational standing to make quick authoritative decisions on appeal by the Data Services Lead should be designated.
+
+Note that while in operations there is a plan for a Data Release board, it is still necessary to have a person (perhaps the same person would would chair this board) that can make timely decisions without a formal process.
+
+### Insufficient internal red-team testing
+
+One of the reasons that it was unclear how close to the deadline we were going to cut it until it was almost too late was the very late involvement of "critical" testers.
+Organisationally there are two groups that can provide pre-release testing, however the V&V team focuses on pre-handover pipeline product testing, and the CST team focus on "best practice" testing for their educational materials.
+
+The majority of late (2-3 week pre-release) serious issues was uncovered by testers with high levels of data and software domain expertise -- Product Owners, Pipeline Developers, Middleware Developers.
```

Comment:
This again was exacerbated by the lack of "pipelining" and ability to internally preview the expected data before the "bucket brigade" handoff from one team to the next.